deliverables/README.md

```diff
@@ -12,19 +16,42 @@ Sign Up for Cloud9 and create a basic Rails project.
 ####Required
 
 What is an "interpreted" language?<br>
+A programming language for which most of its implementations execute instructions directly, without touchin a compiler. Thus the interpreter executes the program through each statment  of one or more subroutines already compiled into machine code. 
+
 What is IRB?<br>
+Stands for "interactive Ruby" and is a tool to interactively execute Ruby expressions read from the standard input. This is usually accessed through the terminal.
+
 What are Objects?<br>
+Objects in programming in real life. Things with specific attributes that make them distinct. Objects are usally an instance of a class or a struct depending on your langugae. 
+
 What are Methods?<br>
+A method is a programmed procedure that is defined as part of a class and included in any object of that class. 
+
 What are Classes?<br>
+Classes are a group of attributes that desbribe a general group of objects. For example a class would be a Bicycle and the attrubtes one would use to distinguish their specific objects might be it's cadence, speed or gear. 
+
 What are Blocks?<br>
+Instructions between curly braces which may optionally take arguments and returns values.
+
 What is an Array?<br>
+Arrays are ordered integer collections of any object or data types.
+
 What is an Iterator?<br>
+An iterator is a method that repeatedly invokes a block of code. 
+
 What are hashes?<br>
+Hashes store related information by giving reusable labels to pieces of our data by using key value pairs. 
+
 What is a library?<br>
+A libraries are a collection of precomplied routines that a program can use.
+
 What is a gem?
+Gems are the libraries for ruby. RubyGems is a packing system designed to facilitate the creation, sharing and installation of libraries. 
 
 Do the challenges at http://tryruby.org/levels/1/challenges/0.
 
+Codecademy Profile: http://www.codecademy.com/nome9478#completed
```

Comment:
This week we're looking for link to codeschool profile:  codeschool.com/users/<your-username>.  But since you finished the codecademy ruby course a while ago don't worry about TryRuby if you haven't already done it.